coordinate.go

```diff
@@ -101,15 +102,15 @@ func (a *Agent) runNodePing(node *api.Node) {
 	// Update the node's health based on the results of the ping.
 	if err == nil {
 		if err := a.updateHealthyNode(node, kvClient, key, kvPair); err != nil {
-			a.logger.Printf("[WARN] error updating node: %v", err)
+			a.logger.Warn("error updating node: ", "error", hclog.Fmt("%v", err))
 		}
 		if err := a.updateNodeCoordinate(node, rtt); err != nil {
-			a.logger.Printf("[WARN] could not update coordinate for node %q: %v", node.Node, err)
+			a.logger.Warn("could not update coordinate for node ", "node", hclog.Fmt("%q: %v", node.Node, err))
```

Comment:
Would you remove the whitespace after "node"? Parameters here could be "node" with "error" separate.